src/NinjaTools.FluentMockServer/MockServerClient.cs

```diff
@@ -91,7 +124,8 @@ public async Task<(bool isValid, string responseMessage)> VerifyAsync([NotNull]
         }
 
         [NotNull]
-        public async Task<HttpResponseMessage> Verify(Action<IFluentVerificationBuilder> verify)
+        [ItemNotNull]
+        public async Task<HttpResponseMessage> Verify([NotNull] Action<IFluentVerificationBuilder> verify)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Rename the parameter 'verify' so that it does not duplicate the method name.](https://app.codacy.com/manual/git_36/NinjaTools.FluentMockServer/pullRequest?prid=4885942)